pkg/apis/v1alpha1/workload.go

```diff
@@ -79,40 +78,10 @@ type WorkloadSpec struct {
 	Resources     *corev1.ResourceRequirements `json:"resources,omitempty"`
 }
 
-type WorkloadSource struct {
-	Git *WorkloadGit `json:"git,omitempty"`
-	// Image is an OCI image is a registry that contains source code
-	Image   *string `json:"image,omitempty"`
-	Subpath *string `json:"subPath,omitempty"`
-}
-
-type WorkloadGit struct {
-	URL *string         `json:"url,omitempty"`
-	Ref *WorkloadGitRef `json:"ref,omitempty"`
-}
-
-type WorkloadGitRef struct {
-	Branch *string `json:"branch,omitempty"`
-	Tag    *string `json:"tag,omitempty"`
-	Commit *string `json:"commit,omitempty"`
-}
-
-type WorkloadParam struct {
-	Name  string               `json:"name"`
-	Value apiextensionsv1.JSON `json:"value"`
-}
-
-type WorkloadSupplyChainReference struct {
-	Kind       string `json:"kind,omitempty"`
-	Namespace  string `json:"namespace,omitempty"`
-	Name       string `json:"name,omitempty"`
-	APIVersion string `json:"apiVersion,omitempty"`
-}
-
 type WorkloadStatus struct {
-	ObservedGeneration int64                        `json:"observedGeneration,omitempty"`
-	Conditions         []metav1.Condition           `json:"conditions,omitempty"`
-	SupplyChainRef     WorkloadSupplyChainReference `json:"supplyChainRef,omitempty"`
+	ObservedGeneration int64              `json:"observedGeneration,omitempty"`
+	Conditions         []metav1.Condition `json:"conditions,omitempty"`
+	SupplyChainRef     OwnerReference     `json:"supplyChainRef,omitempty"`
```

Comment:
hmmm I'm not sure about the name 😬  I'd say it's probably better to go with something like `ObjectReference` (like, `corev1.ObjectReference`) as this is not _really_ an "owner reference" in the sense that the supply chain doesn't _own_ the workload.